examples/experimental/skip_test/primitive_types.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
kinda hard to comment on specific cells in json, but i think some conversion examples would be helpful, can we convert from 1 type to another in weave?

Reply:
great point, added a basic conversion example between String and Number!